src/components/PageHeaderCompact/_page-header-compact.scss

```diff
@@ -72,6 +73,7 @@
     margin-top: calc(var(--body-sm) +
     (2 * var(--space-unit)) +
     (var(--body-line-height) * var(--body-sm) * 0.5));
+    position: absolute;
```

Comment:
What can I say, it works and it's not the first time we've fixed grid gaps using `position: absolute`. I believe all of the following are true so for the time-being it's prob fine (but you should also test this as you mentioned):
* authors and topics are only available in PageHeaderCompact
* PageHeaderCompact is only used on article page types
* InpageNav is not used on article page types

All that said, I have real reservations about using this approach because it is effectively a hack and if it ever came to a situation where we want e.g. authors, topics and in-page nav there is a risk of overlap. Long-term we need to look at how we can refactor page headers, perhaps so that sidebar components are all at the same level and therefore their vertical positions are determined by each other. I also suggest the dev team prototype this for the designers to illustrate the problems we currently have. I'll create a separate ticket.